ui/pages/confirmations/components/confirm/splash/smart-account-update/smart-account-update.tsx

```diff
@@ -0,0 +1,181 @@
+        </Text>
+        <Box display={Display.Flex} alignItems={AlignItems.center}>
+          <Text
+            color={TextColor.textAlternative}
+            variant={TextVariant.bodyMd}
+            marginInlineEnd={2}
+          >
+            {t('smartAccountRequestFor')}
+          </Text>
+          <Name
+            value={from}
+            type={NameType.ETHEREUM_ADDRESS}
+            preferContractSymbol
+            variation={chainId}
+          />
+        </Box>
+        <Box display={Display.Flex} alignItems={AlignItems.flexStart}>
+          <img width="24px" src="./images/speedometer.svg" />
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            marginInlineStart={2}
+          >
+            <Text
+              color={TextColor.textDefault}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Medium}
+            >
+              {t('smartAccountBetterTransaction')}
+            </Text>
+            <Text
+              color={TextColor.textAlternative}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Normal}
+            >
+              {t('smartAccountBetterTransactionDescription')}
+            </Text>
+          </Box>
+        </Box>
+        <Box display={Display.Flex} alignItems={AlignItems.flexStart}>
+          <img width="24px" src="./images/petrol-pump.svg" />
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            marginInlineStart={2}
+          >
+            <Text
+              color={TextColor.textDefault}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Medium}
+            >
+              {t('smartAccountPayToken')}
+            </Text>
+            <Text
+              color={TextColor.textAlternative}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Normal}
+            >
+              {t('smartAccountPayTokenDescription')}
+            </Text>
+          </Box>
+        </Box>
+        <Box display={Display.Flex} alignItems={AlignItems.flexStart}>
+          <img width="24px" src="./images/sparkle.svg" />
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            marginInlineStart={2}
+          >
+            <Text
+              color={TextColor.textDefault}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Medium}
+            >
+              {t('smartAccountFeatures')}
+            </Text>
+            <Text
+              color={TextColor.textAlternative}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Normal}
+            >
+              {t('smartAccountFeaturesDescription')}
+              <a
+                key="learn_more_link"
+                href={SMART_ACCOUNT_INFO_LINK}
+                rel="noreferrer"
```

Comment:
I think we should use our ButtonLink component with the externalLink property. This sets rel="noopener noreferrer". See ui/components/component-library/button-base/button-base.tsx

noopener:
- Prevents the new page from accessing the window.opener property
- This blocks the new page from potentially manipulating your original window
- Protects against "reverse tabnabbing" attacks where a malicious site could redirect your original page

Reply:
Good point, I made this change